### PR TITLE
Link to Global Scale Demo from Agones Examples page

### PR DESCRIPTION
**Pull request by Kalaiselvi84**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug
> /kind cleanup
> /kind documentation
> /kind feature
> /kind hotfix

/kind cleanup
/kind feature

**What this PR does / Why we need it**:

**Which issue(s) this PR fixes**:

Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: site/content/en/docs/Examples/_index.md [hunk 1]
```diff
@@ -46,7 +46,11 @@ These are all examples of simple game server implementations, that integrate the
...
   aware proxy, built on Google Cloud.
 
-## Integration with Open Match
+## Integrations with other projects
 
 * [googleforgames/space-agon](https://github.com/googleforgames/space-agon) - Space Agon is a demo of Agones and 
   Open Match with a browser based game.
+
+## Global Scale Demo
+
```
Comment (markmandel) [inline site/content/en/docs/Examples/_index.md:55]:
```suggestion
```

Comment (markmandel) [reply site/content/en/docs/Examples/_index.md:55]: Let's  get rid of this header, we can put the entry for Global Scale Demo right below Space Agon.

**Comment by markmandel** [comment]
Ah, you will also need to fix the link in the FAQ page:

```
site/docs/faq/index.html
  hash does not exist --- site/docs/faq/index.html --> /site/docs/examples/#integration-with-open-match
  hash does not exist --- site/docs/faq/index.html --> /site/docs/third-party-content/examples/#integration-with-open-match
```

File: site/content/en/docs/Examples/_index.md (hunk 1 above)
Comment (Kalaiselvi84) [reply site/content/en/docs/Examples/_index.md]: Global multiplayer demo link is added below the Space Agon

**Review by markmandel — APPROVED** [review]

**Pull request merged**